### DISCUSSION TITLE
Create, Find, Select suitable images and artworks

### DISCUSSION THREAD

**Discussion by Pradeepkumara95** [ousl-groupE5-ExpiryInspector/ExpiryInspector#18] [Ideas]
we need certain artworks for authentication screens. and good quality images with copyrights for the dashboard and image buttons.

- [ ] sign in page
- [ ] login page
- [ ] forgot password
- [ ] landing page
- [ ] user profile testing
- [ ] item detail page testing
- [ ] dashboard screen
- [ ]  place holder image artwork

**Comment by Pradeepkumara95** [comment]
@Sahani13 Follow this

**Reply by Sahani13** [reply]
![Screenshot 2024-09-17 172141](https://github.com/user-attachments/assets/5c0b3f9a-3ce4-4e2d-a80c-509dbcbd2094)

These are the images that i have created from using AI image generators.

**Comment by Pradeepkumara95** [comment]
@sarasikarunanayaka please review these images, @Sahani13 find suitable images for the dashboard cover, button images and placeholders

**Reply by sarasikarunanayaka** [reply]
I designed these images for the Expiry Inspector app logo.

![Expiry Inspector app logo 1](https://github.com/user-attachments/assets/d6367f69-0725-47f2-aa1b-1759768e1995)
![Expiry Inspector app logo 2](https://github.com/user-attachments/assets/4f5a57f1-15d2-4e9b-85ab-c2b82fb7a5a8)

**Reply by sarasikarunanayaka** [reply]
- I updated logo designs for app because  of the low resolution and also I changed the design of the logo.

**Dashboard logo**
![DashboardLogo](https://github.com/user-attachments/assets/29ab49d7-416f-4228-aaa5-fbae137487cd)

**Updated logo icons**
![logo Icon_Updated](https://github.com/user-attachments/assets/7d3f7070-0f48-41b9-a2ef-a326f9ef2e11)  

![Logo_Updated](https://github.com/user-attachments/assets/aecc70b4-8523-4f36-9e1b-0afa89990d8e)
